CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 1.4.0
+
+* Updated to use Unicode 15.1.0.
+
 ## 1.3.1-wip
```

Comment:
Don't bump to a new version if the previous version has not bee published yet.

Also: does this affect any runtime behavior? if not, no version delta is needed!

Reply:
OK. Do you want to cherry pick the commit that updated the Unicode data? Or should i remove the version changes commits and force push?

I just ran the generate script and not really sure how It will affect runtime behavior but the latest spec introduces new emoji data so could potentially change how you iterate some new emoji characters (?)

Reply:
It doesn't look like any library code has changed!